assets/dev/scss/frontend/widgets/divider.scss

```diff
@@ -137,11 +137,28 @@
 	direction: rtl;
 }
 
-.e-container {
+.elementor {
 
-	& > .elementor-widget-divider {
-		// Fix - Divider widget has undefined width and height issues inside the flex container (ED-7722).
-		width: var( --container-widget-width, 100% );
-		flex-grow: 1;
+	.elementor-element {
```

Comment:
```suggestion
	&-element {
```

Reply:
@StuartElementor That option isn't possible because it will write '.elementor-element' instead of '.elementor .elementor-element', which we need for specificity reasons. 

I have changed my base selector though, for clarity.